### PR TITLE
Updates search and complaint links

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Adds `/complaint/` URL to submit a complaint CTA. Note: URL is not currently active on our testing servers. CC @rosskarchner 
- Adds URL and hidden inputs for hooking up global search to our existing search engine.

## Testing

- Global search should expand, contract, and clear as expected and entering text and clicking search or pressing enter should send you to the search portal.

## Review

- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim 

## Screenshots

screen shot 2016-02-25 at 5 52 00 pm

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/global-search.html [hunk 1]
```diff
@@ -17,21 +17,32 @@
...
     </button>
     <div class="m-global-search_content" aria-expanded="false">
-        <form class="m-global-search_content-form">
+        <form class="m-global-search_content-form"
+              action="http://search.consumerfinance.gov/search"
+              method="get">
+            {# Required hidden input for CFPB search. #}
+            <input type="hidden"
+                           name="utf8"
+                           value="✓">
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/molecules/global-search.html]: The value is really "✓"?

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
:+1:

File: cfgov/jinja2/v1/_includes/molecules/global-search.html (hunk 1 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-search.html]: My guess is that's how it knows it's UTF-8. If that doesn't make it through, it's not.

Comment (sebworks) [inline cfgov/jinja2/v1/_includes/molecules/global-search.html]: http://stackoverflow.com/questions/27338154/why-do-some-websites-have-utf8-in-their-title

Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-search.html]: Nice! Sums it up.

**Pull request merged**